src/pybind/mgr/influx/module.py

```diff
@@ -207,8 +248,10 @@ def send_to_influx(self):
         # instead we'll catch the not found exception and inform the user if
         # db can not be created
         try:
-            client.write_points(self.get_df_stats(), 'ms')
+            df_stats = self.get_df_stats()
```

Comment:
To avoid the [0],[1] indexing in later lines, you could rewrite this one as `df_stats, pools = self.get_df_stats()` and then refer to the results directly by name